reference/protocol.ksy

```diff
@@ -55,11 +54,10 @@ enums:
     0x06: bulk_pull
     0x07: bulk_push
     0x08: frontier_req
-    0x09: bulk_pull_blocks
-    0x0a: node_id_handshake
-    0x0b: bulk_pull_account
-    0x0c: telemetry_req
-    0x0d: telemetry_ack
+    0x09: node_id_handshake
```

Comment:
node_id_handshake must remain unchanged at value 0xA.
Same thing applies for the others below.

Reply:
Thanks, resolved!